Modules/BenchPress.Azure/Public/CustomAssertions/BeInLocation.ps1

```diff
@@ -31,12 +31,21 @@ function ShouldBeInLocation ($ActualValue, [string]$ExpectedValue, [switch] $Neg
     if ($Negate) { $succeeded = -not $succeeded }
     $failureMessage = "ConfirmResult is null or empty."
   }
+  elseif (-not $ActualValue.ResourceDetails.Location) {
+    [bool] $succeeded = $false
+
+    if ($Negate) { $succeeded = -not $succeeded }
```

Comment:
Yeah. The current implementation is basically giving the ability to do `ErrorAction -SlientlyContinue` through `-Not`. 
The case above is probably more suited for `-Not -BeDeployed`.

I like the idea of having the option to say `-Not` and still have it pass. Okay will both. Will bring it up in standup

Reply:
In my opinion it is an invalid test and should fail if there is no location property and the user is testing for a location. Even in a negative situation, because you can't prove the negative unless you have the property.